.github/workflows/semantic_release.yml

```diff
@@ -18,11 +18,17 @@ jobs:
         with:
           node-version: "14"
       - name: Setup node deps
-        run: npm i conventional-changelog-conventionalcommits@4 -D
+        run: |
+          npm i conventional-changelog-conventionalcommits@4 -D
+          npm run build
```

Comment:
If you are trying to build the doc here, i'm not sure you are in the correct directory... 
I think you can use `--prefix` to specify the directory. Or just cd in it.

Reply:
You are right, it was not in the correct directory. This should be fixed now